changelog.md

```diff
@@ -1,3 +1,121 @@
+
+# CITYENGINE SDK 3.3.11173 CHANGELOG
+
+This section lists changes compared to CityEngine SDK 3.2.10824.
+
+## General Info
+* CityEngine SDK 3.3.11173 is used in CityEngine 2025.0.11173.
+
+## PRT API
+* `prt::InitalShapeBuilder::setAttributes()`: support for disabling the numerical geometry cleanup .
+* `prt::generate()`: Fixed a small mem leak when CGA imports were used.
+* `prt::createRulFileInfo()` Imporoved perfomance for cgb files with many imports.
+
+## PRTX API
+* Added `prtx::MeshUtils::transformNormals()` function.
+
+## CGA
+* Changes to existing features:
+    * union, subtract, intersect operations: The syntax was modified to explicitly state both operands. The previous syntax is now deprecated. A switch to the new syntax can be done as follows for the default mode and swap mode:
+        * subtract Rule => subtract { OperandA. | Rule }
+        The current shape is explicitly stated with a shape symbol serving as operand A.
```

Comment:
"The current shape ... " is not rendered in a new line... looks weird
see
https://github.com/Esri/cityengine-sdk/blob/matt6768/CE-13023_changelog/changelog.md

you could also just remove "A switch to the new syntax can ... " and everything below

Reply:
fixed it